superglue/lib/utils/ujs.ts

```diff
@@ -107,10 +111,15 @@ export class HandlerBuilder {
       target = this.remote
     }
 
+    if (linkOrForm.getAttribute(this.attributePrefix + '-replace')) {
+      target = this.visit
+      suggestedAction = 'replace'
+    }
+
```

Comment:
So some of this change can be moved to between line 107 and 108 instead since its an option that only affects `visit`. It could look something like `opts.suggestedAction = "replace"`